hypothesis-python/tests/cover/test_reproduce_failure.py

```diff
@@ -48,6 +48,8 @@ def test_decoding_may_fail(t):
         reject()
     except InvalidArgument:
         pass
+    except Exception as e:
```

Comment:
I’m mildly interested that this doesn’t need an “except: pass”. Do we not run coverage in tests?

(Should we?)

Reply:
I *think* you mean a `pragma: no cover`, because we fail if the branch is not taken?

We don't currently check coverage of our tests - I guess we could, but it would probably be expensive to run all of them and I don't see much benefit.

Reply:
We used to. The main reason why we don't is that there were a bunch of tests where we have functions that we never run and it was kinda annoying to annotate them all. I've no strong preference either way.